src/app/data-planes/components/data-plane-traffic/ServiceTrafficCard.vue

```diff
@@ -110,5 +125,11 @@ const props = defineProps<{
   display: flex;
   flex: 1 1 auto;
 }
-
+.title :deep(> a) {
+  text-decoration: none;
+  color: inherit;
+}
+.data-card:has(a) {
```

Comment:
**Question**: Should we be using `:has` already? It’s a really new feature that is only supported in Firefox as of version 121 which is the current stable release.

Reply:
Well I'm glad you asked! This only adds a pointer cursor instead of the default. If it was something more visible/important I wouldn't have used it.

I think more generally, I suppose it depends on what our browser support is? It would be good to figure out some sort of rule to follow.

Safari's support is from May last year (which also introduced `Array.at`), Chrome's from September last year. Firefox is the most recent (but also I would guess less used) at December only a month back.

Back to the less general issue of using `:has` to change the cursor to a pointer, I'm not too concerned, but would be fine to change it to use a different approach instead, lmk.

Reply:
It would be better to control this via a prop on DataCard. `.data-card:has(a)` isn’t great: it relies on a link in a data card implying that it is linked up. But what if there are other links in it? Then this selector would match for cards that aren’t linked up like that.

On the matter of browser support, _a month old_ is definitely not old enough to consider the feature well-supported. We shouldn’t use a case-by-case heuristic á la "we can use more modern features for things that aren’t so critical". That just makes the decision making very messy.

Reply:
Ok no prob, I moved this higher up (its only the passthrough card that doesn't link/need a pointer so I added the pointer based on that)

---

> But what if there are other links in it?

There never are, it was only to exclude the passthrough in `ServiceTrafficCard` which is only used here.

> On the matter of browser support, a month old is definitely not old enough to consider the feature well-supported.

Its a month old for Firefox (which is far from the only browser you take into account when discussing 'browser support'), which at a rough count/guess is probably less than 10% of users I'd say 90% of those Firefox users are on the the last latest version (120) so are likely to be on the latest (121) in the next few weeks.

For that very small amount of users, they will just not see a pointer cursor in this currently feature flagged new feature unless they rollover the title, just for the next few weeks.

Granted there will always be the odd person that will have an older browser for the next few years, but thats what graceful degradation is all about, the functionality isn't completely broken, its just not as nice as if you had an up-to-date browser.

Again, more than happy to discuss and decide upon a browser support matrix for all browsers, in fact I think we should do that, but it should include graceful degradation exceptions.

Anyway, I'm resolving this seeing as I moved the rule, please reopen if you want anything more here.